step2/hackatom-stash-controller.sh

```diff
@@ -1,79 +1,106 @@
 #!/usr/bin/env bash
 set -e
 
-# This script is for setting up the stash and controller accounts.
+# This script is for setting up the stash and controller accounts, and sending
+# some txs from controller on behalf of stash. All commands are commented.
 
 # Set variables
 CFG_DIR=~/.simapp
-BUILD_CMD=./cosmos-sdk/build/simd
+BUILD_CMD=../step1/cosmos-sdk/build/simd
```

Comment:
@likhita-809 this is correct? I was running:
```
cd step2
./hackatom-stash-controller.sh
```
so i needed this

Reply:
If we are are cloning sdk from step1 directory, this is needed.